pattern/equivalence/AlphaEquivalence.java

```diff
@@ -113,9 +118,24 @@ public Valid asValid() {
             return this;
         }
 
-        HashMap<Variable, Variable> map() {
+        public HashMap<Variable, Variable> variableMapping() {
             return new HashMap<>(map);
```

Comment:
don't we have a helper `map()` function for this? hmmm

Reply:
I don't understand - this is that helper method, but called `variableMapping()` now to be explicit as to which map you'll get back.